lib/client.js

```diff
@@ -17,6 +17,7 @@ const uuid = require("uuid/v1");
 const { AdobeAuth } = require("@adobe/asset-compute-events-client");
 const { AssetCompute } = require("./assetcompute");
 const { AssetComputeEventEmitter } = require("./eventemitter");
+const { AdobeIOEvents } = require("@adobe/asset-compute-events-client");
```

Comment:
You can merge this with the require L17. This will give you:
const { AdobeAuth`, AdobeIOEvents  } = require("@adobe/asset-compute-events-client");

Reply:
thanks!